modules/nf-core/grabix/check/main.nf

```diff
@@ -0,0 +1,30 @@
+process GRABIX_CHECK {
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/grabix:0.1.8--hdcf5f25_9':
+        'biocontainers/grabix:0.1.8--hdcf5f25_9' }"
+
+    input:
+    path(input)
+
+    output:
+    env COMPRESS_BGZIP  , emit: compress_bgzip
```

Comment:
```suggestion
    tuple val(meta), env(COMPRESS_BGZIP)  , emit: compress_bgzip
```
So you output with the same meta as the input to be able to see for what kind of data you just ran the process